modules/ank/README.md

```diff
@@ -0,0 +1,73 @@
+# Λnk
+
+_Compile time docs verification and evaluation for Kotlin and Java._
+
+_Λnk_ short for [Ankhesenpaaten](https://en.wikipedia.org/wiki/Ankhesenamun) wife and sister of [Tutankhamun](https://en.wikipedia.org/wiki/Tutankhamun), is a [Gradle](https://gradle.org/) plugin to verify code snippets in library docs for the Kotlin and Java programming languages.
+
+Λnk is inspired by the awesome docs and tutorial generator for Scala [`tut`](http://tpolecat.github.io/tut/).
+
+Λnk is a very simple documentation tool for Kotlin and Java written using [Λrrow](https://github.com/arrow-kt/arrow) that reads Markdown files and interprets and evaluates Kotlin and Java code in `ank` sheds, allowing you to write documentation that is typechecked and run as part of your build.
+In a nutshell, Λnk works evaluating, then capturing results and including them after the expressions so they can be read in the documentation. The point of Λnk is to provide code that the user can type in and expect to work. 
+
+## Basic setup
+
+In your root `build.gradle`, add the Gradle Λnk dependency:
+
+```groovy
+buildscript {
+
+    repositories {
+        // ...
+        maven { url "https://dl.bintray.com/arrow-kt/arrow-kt/" }
+    }
+    
+    dependencies {
+        // ...
+        classpath 'io.arrow-kt:arrow-ank-gradle:<version>'
+    }
+}
+```
+
+Then apply it to each individual module where you want to use Λnk:
+
+```groovy
+apply plugin: 'ank-gradle-plugin'
+
+dependencies {
+    implementation 'io.arrow-kt:arrow-ank:<version>'
+    // ...
+}
+
+ank {
+    source = file("src/main/docs")
+    target = file("build/site")
+    classpath = sourceSets.main.runtimeClasspath
+}
+```
+
+## Run Λnk
+
+```
+./gradlew :module-name:runAnk
+```
+
+This will process all `*.md` files in `source`, write them to `target`, while providing all runtime dependencies (`sourceSets.main.runtimeClasspath`) in the `classpath`.
+
+## Errors output
+
+When something goes wrong Λnk shows what snippet has failed and where, including the compiler errors.
```

Comment:
This should probably show an example with a snippet that fails and how ank reports it.